### DISCUSSION TITLE
RS485 for Optiboot

### DISCUSSION THREAD

**Discussion by expertup** [SpenceKonde/ATTinyCore#694] [Support / Q & A]
I want to compile optiboot for attiny85 with rs485. I need and control pin to PB2. While do MAKE, I have this error:

![optiboot.PNG](https://user-images.githubusercontent.com/1708687/172876753-6b676a99-3a81-4c8e-bd3b-9a3265eadef8.PNG)

How I can get Optiboot with support MAX485 on PB2 pin?

**Comment by SpenceKonde** [comment]
You do a .zip install of arduino 1.0.6, delete everything but hardware and tools, copy the attinycore bootloader folder into there, (location/hardware/arduino/bootloader), copy the latest toolchain on top of the old one then run makealltiny.bat
